content/contributing-terms.en.md

```diff
@@ -319,6 +323,74 @@ export async function convertImagesToBase64(document, documentDeclaration) {
 }
 ```
 
+#### Terms
+
+Now that source documents are defined let's see how to finally declare terms. 
+
+In the case where terms are consituted by only one source document, the declaration of the terms is done by simply declaring the terms as it _was_ the associated source document:
+
+```json
+  …
+  "documents": {
+    "<terms type>": {
+      "fetch": "…",
+      "executeClientScripts": "…",
+      "filter": "…",
+      "remove": "…",
+      "select": "…"
+    }
+  }
+  …
+```
+
+When the terms are distributed across multiple source documents, it should be declared by defining the combination, with `combine`, of theses multiples source documents:
+
+```json
+  …
+  "documents": {
+    "<terms type>": 
+      "combine": [
+        {
+          "fetch": "…",
+          "executeClientScripts": "…",
+          "filter": "…",
+          "remove": "…",
+          "select": "…"
+        },
+        {
+          "fetch": "…",
+          "executeClientScripts": "…",
+          "filter": "…",
+          "remove": "…",
+          "select": "…"
+        }
+      ]
+  }
+  …
+```
+
+Note that if some elements of source documents can be factorized (for example, it is common for the structure of HTML pages to be similar from page to page, so `select`, `remove` and `filter` can be factorized) it can be done like this:
```

Comment:
```suggestion
If some parts of the source documents are repeated, they can be factorised. For example, it is common for the structure of HTML pages to be similar from page to page, so `select`, `remove` and `filter` would be the same. These elements can be shared instead of being duplicated:
```